content/docs/reference/collections.md

```diff
@@ -70,3 +70,36 @@ $localization:
 ### categories
 
 The `categories` key contains a list of categories that documents inside the collection can fall into. A collection's categories can be used in conjunction with the `g.categories` template function to iterate over the documents in the collection, grouped by category.
+
+## Nested collections
```

Comment:
One thing that could be clarified is that sub directories do not need to have their own `_collection.yaml` file. 

For instance, if your `/content/pages/_blueprint.yaml` has a path of `/{root}/{collection.sub_path}/{base}` then a `/content/pages/books/about.yaml` would have the path of `/books/about/` without having to declare a new `_blueprint.yaml` in the `books` directory.

Reply:
Thanks, I didn't understand that. I tried to explain it, have a look if it's understandable. Thanks!